en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+        ```
+        jdbc:db2://localhost:50000/idndb
+        ```
+    ??? tip "Sample configuration written for PostgreSQL DB type URL"
+        
+        ```
+        jdbc:postgresql://localhost:5432/idndb
+        ```
+    ??? tip "Sample configuration written for MSSQL DB type URL"
+        
+        ```
+        jdbc:sqlserver://localhost:1433;databaseName=idndb
+        ```
+    ??? tip "Sample configuration written for Oracle DB type URL"
+        
+        ```
+        jdbc:oracle:thin:@localhost:1521/ORCLCDB.LOCALDOMAIN
+        ```
+
+6.  Run the tool using `./keyrotation.sh keyrotation-tool-<version>-SNAPSHOT.jar properties.yaml` command from  inside the `<KEY_ROTATION_TOOL>` location
+
+7.  Edit the `<NEW_IS_HOME>` `deployment.toml` file having the new configured databases and the new key 
+
+8.  Start the `<NEW_IS_HOME>` once existing DB and config file data re-encryption completes
+
+9.  When no new entries are being synced in the logs, route traffic to the `<NEW_IS_HOME>` and enable all load balancer API endpoints(privileged and end user flows)
+
+!!! note
+    Do not stop the tool at once, let it sync any remaining data in the temp tables after routing the traffic
+
+## Verifying the key rotation
+
+*   Check the log files to verify if re-encryption happened successfully for the 7 identity and registry database tables. Check below logs for successful/failed re-encryption counts of OAuth2 access and refresh tokens
+
+    ```tab="DB log sample"
+    Successfully updated OAuth2 access and refresh tokens data records in IDN_OAUTH2_ACCESS_TOKEN: 897
+    Failed OAuth2 access and refresh tokens data records in IDN_OAUTH2_ACCESS_TOKEN: 0
+    ```
+
+!!! note
+    If the key rotation task is successful there should be 0 as the failed logs count for all the tables
+
+*   Check the log files to verify if re-encryption happened successfully for the 3 configuration files. Check below logs for successful/failed re-encryption counts of event publisher configuration files
+
+    ```tab="Config file log sample"
+    Updated event publisher configuration files: 8
+    Failed event publisher configuration files: 0
+    ```
+
+!!! note
+    If the key rotation task is successful there should be 0 as the failed logs count for all the configuration files
+
+*   Check the log files to verify if transformation of the synced data happened successfully for the 4 tables. Check below logs for successful/failed transformation of `IDN_IDENTITY_USER_DATA` table
+
+    ```tab="Synced log sample"
+    Successfully transformed totp data records in IDN_IDENTITY_USER_DATA_TEMP: 2
+    Transformation failed totp data records in IDN_IDENTITY_USER_DATA_TEMP: 0
+    ```
+
+!!! note
+    If the synced task is successful there should be 0 as the failed logs count for all the synced tables
+
+*   Check for any errors in the log files and carefully analyse through the logs, if the error can be ignored proceed with other steps otherwise check what has caused the error.
```

Comment:
```suggestion
*   Check for any errors in the log files and carefully analyze the logs if the error can be ignored and other steps can be proceeded with. Otherwise, check what has caused the error.
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808